src/actions/tests/expressions.spec.js

```diff
@@ -60,11 +60,12 @@ describe("expressions", () => {
   it("should evaluate expressions global scope", async () => {
     const { dispatch, getState } = createStore(mockThreadClient);
 
-    dispatch(actions.addExpression("foo"));
-    dispatch(actions.addExpression("bar"));
+    await dispatch(actions.addExpression("foo"));
+    await dispatch(actions.addExpression("bar"));
+
+    expect(selectors.getExpression(getState(), "foo").value).toBe("bla");
+    expect(selectors.getExpression(getState(), "bar").value).toBe("bla");
```

Comment:
@jasonLaster i'll love your a quick glance on these tests. i'm not feeling they do what i think they should